WebSocketTextClient/WebSocketTextClient.cs

```diff
@@ -123,7 +123,7 @@ public Task SendAsync(string str)
             }
             catch (Exception ex)
             {
-                this.ErrorReceived?.Invoke(this, new SocketErrorEventArgs { Exception = ex, Message = string.Empty });
+                this.ErrorReceived?.Invoke(this, new SocketErrorEventArgs { Exception = ex, Message = "An error occoured." });
```

Comment:
Yet another typo

Reply:
Actually, i don't see any value in SocketErrorEventArgs.Message field. We Already have error description inside Exception property. Can you remove it?

Reply:
You usually set the message property to a human readable message, since sometimes you are not getting the correct exception, although afaik, this is usually encountered in WPF.

I can remove it, sure.

Reply:
Yes, remove it please.
I guess error message for end user should be set in hosting application, not in the library. This way we can handle localization, etc.
Not sure about WPF, i use it in console app/windows service

Reply:
I want to add, that the logging library of netcore (Microsoft.Extensions.Logging) the log warning and log critical methods have the following header: `LogWarning(Exception exception, string message, params string[] args);` and `LogCritical(Exception exception, string message, params string[] args);`